### PR TITLE
Add link to user community on Google+

### PR DESCRIPTION
**Pull request by ImEmJay**
Description:

Adds a link to the new [AlexaPi Users](https://plus.google.com/communities/105607055053826225738/) community on G+.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,5 @@ If your AlexaPi isn't running on startup, crashes or your audio input / output i
 - our **[Documentation Wiki](https://github.com/alexa-pi/AlexaPi/wiki)** - the sections with _debugging_ in their name are your friends!
 - our **[Issue Tracker](https://github.com/alexa-pi/AlexaPi/issues)** 
 
-or chat with us at **[gitter.im/alexa-pi/Lobby](https://gitter.im/alexa-pi/Lobby)**.
\ No newline at end of file
+or chat with us at **[gitter.im/alexa-pi/Lobby](https://gitter.im/alexa-pi/Lobby)**  
+or join our **[AlexaPi Users](https://plus.google.com/communities/105607055053826225738/)** user community on **[Google+](https://plus.google.com)**.
```
Comment (renekliment) [inline README.md]: Hm, do we want to link to G+ itself? Doesn't really matter, just a thought. 

And maybe we should write
```
Also, you can
- chat with us at ...
- join our ...
```
I'm not sure _or_ made sense here in the first place.

Could you please do the small update?

**Comment by ImEmJay** [comment]
@renekliment Done

**Comment by renekliment** [comment]
Merging.

**Comment by ImEmJay** [comment]
Thanks for merging.

**Pull request merged**